Patch-Source: https://github.com/fwupd/fwupd/commit/3147fef735330fba6bd280faa829a6a952bfa997
From 3147fef735330fba6bd280faa829a6a952bfa997 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Thu, 4 Nov 2021 15:38:27 +0000
Subject: [PATCH] uefi-capsule: use /bin/sh in grub hook

Use . instead of source as source is a bashism.

Use /bin/sh as the interpretter as there are no strict bashisms in the
hook.

Ideally the shell expansion using `` is changed to $(), but the escaping
behaviour is complex and dash/busybox sh both support ``.
---
 plugins/uefi-capsule/fwupd.grub.conf.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/uefi-capsule/fwupd.grub.conf.in b/plugins/uefi-capsule/fwupd.grub.conf.in
index 92bd49a1c..9c3a22f23 100755
--- a/plugins/uefi-capsule/fwupd.grub.conf.in
+++ b/plugins/uefi-capsule/fwupd.grub.conf.in
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/sh
 # SPDX-License-Identifier: LGPL-2.1+
 set -e
 
@@ -8,7 +8,7 @@ set -e
 
 if [ -f @localstatedir@/lib/fwupd/uefi_capsule.conf ] &&
    ls /sys/firmware/efi/efivars/fwupd-*-0abba7dc-e516-4167-bbf5-4d9d1c739416 1>/dev/null 2>&1; then
-      source @localstatedir@/lib/fwupd/uefi_capsule.conf
+      . @localstatedir@/lib/fwupd/uefi_capsule.conf
       if [ "${EFI_PATH}" != "" ] && [ "${ESP}" != "" ]; then
       echo "Adding Linux Firmware Updater entry" >&2
 cat << EOF
